content/_index.md

```diff
@@ -36,6 +36,14 @@ for enrollment details!
     alt="Nia Bailey standing in front of the Public Library in New York"
     caption="Nia Bailey completed the UI/UX Designer track in 2021." >}}
 
+{{% if-equal enroll false %}}
+# Do you share our mission to help underrepresented minorities get access to training so that they can build careers in tech?
+{{% /if-equal %}}
```

Comment:
These don't have to be two separate if-equal blocks - you can combine them since they go together (remove line 41 and 43)